.env.example

```diff
@@ -0,0 +1 @@
+LOGGING_URL=
```

Comment:
Can we have this in [`contrib/.env.sample`](https://github.com/okfn-brasil/serenata-de-amor/blob/master/contrib/.env.sample) instead?

Reply:
done

Reply:
Hum… actually I might have not expressed myself properly. Do we need two `.env.sample` files? One in root (added by this PR) **and**  the other one already included in `contrib/` directory?

IMHO either we:

* add the line `LOGGING_URL=` to the existing `contrib/.env.sample`;
* **or** we move the existing `contrib/.env.sample` to root and merge it to the `.env.sample`/`.env.example` included in this PR.

So we end up with _only one_ `.env.sample`. Does that make sense?

Reply:
My bad. Can you check now, after the changes?